swap_meet/item.py

```diff
@@ -1,2 +1,21 @@
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, category="", condition=0):
+        self.category = category
+        self.condition = condition
+    
+    def __str__(self):
+        return "Hello World!"
+
+    def condition_description(self):
+        if not self.condition:
+            return "This item has been heavily used."
+        elif self.condition == 1:
+            return "This item has been moderately used."
+        elif self.condition == 2:
+            return "This item has been partially used."
+        elif self.condition == 3:
+            return "This item is in good condition."
+        elif self.condition == 4:
+            return "This item is in great condition."
+        elif self.condition == 5:
+            return "This item is brand new!"
```

Comment:
Nice handling for condition! It looks like the code assumes that `self.condition` will always be an integer, but what if we allowed decimals like a condition of `3.5`? How could we adapt our code to convert the decimal to an int we could use here, or alternately, change the if-statements to support decimals?

Another approach for finding the condition description could be to make a dictionary that maps conditions as keys to descriptions as values. There's a trade off of using more memory creating the map variable, but if memory isn't a concern, for me, it's a little easier to read.
```suggestion
        condition_map = {
            0: "This item has been heavily used.",
            1: "This item has been moderately used.",
            2: "This item has been partially used.",
            3: "This item is in good condition.",
            4: "This item is in great condition.",
            5: "This item is brand new!"
        }
        
        return condition_map[self.condition]
```